trade-finance-manager-ui/templates/index.njk

```diff
@@ -51,11 +51,12 @@
 
     {% include "_partials/footer.njk" %}
 
-    <script src="/assets/js/main.js" type="text/javascript" integrity="sha512-dAKNB/hxAScYffs+bGUzzx7n48zYFvQRScBqQv/hFOtxplbN0BwnOaS+QuyE24WJXUbr9BQpR1xsJNxwldJBuw==" crossorigin="anonymous"></script>
+    <script src="/assets/js/main.js" type="text/javascript" integrity="sha512-BLhltBWOGogiNck88Bgk9ZP60WzkF0M4UDKNXASvMQ0XBcLRtMCqs77xMkkL+crGcCqpKtuOkkeuTvpTQFuzIA==" crossorigin="anonymous"></script>
     <script src="/assets/js/govukFrontend.js" type="text/javascript" integrity="sha512-BlSKFkwy/7rAHvMRdTX1Kdk/6KgF2exLAH18xCPxd2ukztWA9adhCM/PczucxxNq29j8gvfAaEcGmgg3BbJufA==" crossorigin="anonymous"></script>
     <script src="/assets/js/mojFrontend.js" type="text/javascript" integrity="sha512-VJYfsojYZY4stpSFP22EQ/CXqqrW9BWYWKaiEnYGmyw22rggV4DCe+Lzn8L7Gt+bRq3AyWuNYFadfrCmao0o+A==" crossorigin="anonymous"></script>
     <script src="/assets/js/disableFormSubmitOnSubmission.js" type="text/javascript" integrity="sha512-tCINlYTcYERel2bdysTMl5KaemDdEyog5JSS3hYQqHS6ZIyT4ATLu4oTPerbZ+vAOn4AAzDEsRBSQVacGhoieQ==" crossorigin="anonymous"></script>
     <script src="/assets/js/enableSelectAllTableCheckbox.js" type="text/javascript" integrity="sha512-ER/5raNAJX5VTcyrRBjVExC/f39WaVY4r8k1CyGqJUad95QZ0A1H6zHtsFpdf0JDBLV0acXzSA1B2mkKC8g5Tg==" crossorigin="anonymous"></script>
-  <script src="/assets/js/enableFindReportsByYearDropdown.js" integrity="sha512-QLvf2zIp3WAaUOyTIf4SlZ9JLBWVw1CCwwY8Ozzc3+mTsFv4WClwZ9ZYNdNwf9nSFNd2Y4WNr1GL/Bc6ABo1Fw==" type="text/javascript" crossorigin="anonymous"></script>
+    <script src="/assets/js/enableFindReportsByYearDropdown.js" integrity="sha512-QLvf2zIp3WAaUOyTIf4SlZ9JLBWVw1CCwwY8Ozzc3+mTsFv4WClwZ9ZYNdNwf9nSFNd2Y4WNr1GL/Bc6ABo1Fw==" type="text/javascript" crossorigin="anonymous"></script>
+    <script src="/assets/js/mojFilterHide.js" type="text/javascript" integrity="sha512-SopYqjSHGoALKGfzoUMnm3ovAiTAQP0zz9UQPgo8N+IGD9XPscbhKGWg0fQ5txgzrfm2kURyK9D5iKalsMLwMA==" crossorigin="anonymous"></script>
```

Comment:
Are we able to include this only in the page that requires this script? Otherwise every page will be loading this resource.

Appreciate the other scripts aren't part of this PR, but we should be doing the same with e.g the following:

```
enableSelectAllTableCheckbox
enableFindReportsByYearDropdown
```

FYI @Zainzzkk

Reply:
Moved these 'single-use' scripts into their respective templates (tried to work this out from the PRs they were merged in with) - had to pull the mojFrontend.js depdency up to the top of the body though as some of these scripts are dependent on this being included first